### ISSUE TITLE
getErrors function in DataSource interface

### ISSUE THREAD

**Issue by solleer** [Level-2/Maphper#50]
The `getErrors` function in the DataSource interface currently returns nothing but an empty array in all implementations of it. I feel like the implementations should be fixed or the function removed from the interface.

**Comment by solleer** [comment]
Currently the function is not used by Maphper

**Comment by TomBZombie** [comment]
Yes, you're right. It's probably best to let exceptions happen than to catch them and force them into an array
